cmd/ike/config/env_vars.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"github.com/maistra/istio-workspace/pkg/openshift/parser"
+
+	openshiftApi "github.com/openshift/api/template/v1"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// These consts are used across the project to load corresponding env variables
+// Extend when needed
+const (
+	EnvDockerRepository = "IKE_DOCKER_REPOSITORY"
+	EnvDockerRegistry   = "IKE_DOCKER_REGISTRY"
+)
+
+var (
+	log        = logf.Log.WithName("session_handler")
+	parameters []openshiftApi.Parameter
+)
+
+func init() {
+	collectTplParams("operator.yaml")
```

Comment:
For future safety; iterate over all Assets?

Reply:
Ace! ace0ee7